service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -164,6 +169,26 @@ private JsonObject erLookup(final JsonObject bodyJson, Boolean failIfMultipleFou
                     String responseBody = response.getBody();
                     ids = ERLookupController.getIdsfromResponseBody(responseBody);
 
+                    // Checking ER lookup has options field present or not
+                    if (lookupLinks.get(i).toString().contains("options")) {
+                        final JsonObject lookup = lookupLinks.get(i).getAsJsonObject().get("%lookup%")
+                                .getAsJsonObject();
+                        for (Entry<String, JsonElement> options : lookup.get("options").getAsJsonObject().entrySet()) {
+                            final String optionKey = options.getKey();
+                            final String optionValue = options.getValue().getAsString();
+                            if (optionKey.equals("failIfNoneFound") && (optionValue.equalsIgnoreCase("true")
+                                    || optionValue.equalsIgnoreCase("false"))) {
+                                failIfNoneFound = Boolean.parseBoolean(optionValue);
```

Comment:
what will be the value for second lookup if my first lookup has options and has true for failIfNoneFound, and second doesn't have any options, it should take from the global value instead.

Reply:
the value for second lookup is taking first lookup value instead of global value. I changed the code for this scenario.